docs/approvals/index.md

```diff
@@ -5,7 +5,10 @@ position: 95
 hideInThisSection: true
 ---
 
-Octopus currently supports blocking deployments for a project and environment based on the approval status of a Change Request in a supported ITSM system. Currently, only ServiceNow is supported.
+Octopus currently supports blocking deployments for a project and environment based on the 
+approval status of a Change Request in a supported ITSM system. Currently, both ServiceNow and 
+Jira Service Desk are supported.
 
-For more information, read our documentation on [Service Now Integration](service-now/index.md).
+For more information, read our documentation on [Service Now Integration](service-now/index.md) 
+and [Jira Service Management Integration](jira-service-management/index.md).
```

Comment:
Removing this allows for the child page selector to show the child pages for you as they expand over time :) 
```suggestion
```